open-apparel-registry/open-apparel-registry
# Revert download format

## Overview

Restores the previous download structure of one facility per row.

Connects #2392 

## Demo

New download format
[facilities - new.csv](https://github.com/open-apparel-registry/open-apparel-registry/files/11534473/facilities.-.new.csv)

Embedded download format
[facilities - embed.csv](https://github.com/open-apparel-registry/open-apparel-registry/files/11534499/facilities.-.embed.csv)

## Testing Instructions

On `develop`:
* Run `./scripts/server`
* Download facilities, ensuring you have facilities with extended fields, claim data, and contributor fields, as well as some private and inactive items. 
* Download facilities in an embedded map (preferably one with contributor fields)

On this branch:
* Run `./scripts/server`
* Download facilities, ensuring you have facilities with extended fields, claim data, and contributor fields, as well as some private and inactive items. 
* Download facilities in an embedded map (preferably one with contributor fields)

* Compare the downloads. The data content should be the same, but the rows should be combined into one row per facility. 

## Checklist
